proxy/stat/server.py

```diff
@@ -6,7 +6,33 @@
 from common.stat.metric import StatRegistry, StatSummary, StatGauge
 from common.stat.prometheus import PrometheusServer
 from common.utils.process_pool import ProcessPool
-from .api import STATISTIC_ENDPOINT, TxPoolData, TxFailData, TxDoneData
+from .api import (
+    OpResourceNeonBalanceData,
+    OpResourceSolBalanceData,
+    STATISTIC_ENDPOINT,
+    TxPoolData,
+    TxFailData,
+    TxDoneData,
+)
+
+
+class OpResourceStatApi(AppDataApi):
+    name: ClassVar[str] = "ProxyStatistic::OpResource"
+
+    def __init__(self, registry: StatRegistry):
+        super().__init__()
+        self._op_res_neon_balance = StatGauge("op_res_neon_balance", "Operator resource NEON balance", registry=registry)
```

Comment:
IMHO, it isn't clear that the balance has different tokens under the hood.
I think it will be better to rename the metric to something like:
-  `op_res_native_balance`
-  OR `op_res_earned_tokens_balance`

Reply:
To be consistent with the client, I called it `op_resource_earned_tokens_balance`.